pkg/playground/patterns.html

```diff
@@ -197,18 +210,364 @@ <h4 class="modal-title" translatable="yes">Example Dialog</h4>
+                                            <dd>fedora-atomic</dd>
+                                            <dt>Revision Number</dt>
+                                            <dd>24.1</dd>
+                                            <dt>Released</dt>
+                                            <dd title="2015-08-29 15:33">1 month ago</dd>
+                                        </dl>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </td>
+            </tr>
+        </tbody>
+        <tbody>
+            <tr>
+                <td>
+                    <div class="timeline">
+                        <div class="timeline-point timeline-md">
+                            <div class="spinner"></div>
+                        </div>
+                        <div class="listing-panel">
+                            <div class="listing-head">
+                                <div class="listing-actions">
+                                </div>
+                                <div class="listing-status">
+                                    Downloading manifests 85%
+                                </div>
+                                <h3>fedora-atomic: 24.0</h3>
+                                <ul class="nav nav-tabs nav-tabs-pf">
+                                    <li class="active"><a href="#">Tree</a></li>
+                                    <li><a href="#">Packages</a></li>
+                                </ul>
+                            </div>
+                            <div class="listing-body">
+                                <div class="row">
+                                    <div class="col-sm-6">
+                                        <dl>
+                                            <dt>Operating System</dt>
+                                            <dd>fedora-atomic</dd>
+                                            <dt>Revision Number</dt>
+                                            <dd>24.1</dd>
+                                            <dt>Released</dt>
+                                            <dd title="2015-08-29 15:33">1 month ago</dd>
+                                        </dl>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </td>
+            </tr>
+        </tbody>
+    </table>
+
+</div>
+
 <script>
 require([
-    'jquery',
-    "base1/bootstrap-select",
+    'jquery'
 ], function($) {
     "use strict";
 
-    $(document).ready(function() {
-        $('.selectpicker').selectpicker();
-        $("body").show();
+    $("body").on("click", ".listing-item, .listing-head", function(ev) {
+        if($(ev.target).parents().andSelf().filter(".btn, a, li, .timeline").length === 0) {
```

Comment:
see previous comment, [andSelf() is deprecated](https://api.jquery.com/andself/)
long chain of commands may merit a comment